src/components/MediatedRequestsActivities/components/MediatedRequestsFilters/MediatedRequestsFilters.js

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+
+import {
+  MediatedRequestLevelFilter,
+  MediatedRequestStatusFilter,
+  Search,
+} from './components';
+
+import {
+  getIsTitleLevelRequestsFeatureEnabled,
+} from '../../../../utils';
+
+const SETTINGS_SCOPES = {
+  CIRCULATION: 'circulation',
+};
+const SETTINGS_KEYS = {
+  GENERAL_TLR: 'generalTlr',
+};
+
+const MediatedRequestsFilters = ({
+  resources,
+}) => {
+  const isTitleLevelRequestsFeatureEnabled = getIsTitleLevelRequestsFeatureEnabled(resources);
+
+  return (
+    <form
+      data-testid="MediatedRequestsFiltersForm"
+    >
+      <Search
+        data-testid="MediatedRequestsFiltersSearch"
+      />
+      <MediatedRequestStatusFilter
+        data-testid="mediatedRequestFiltersStatusFilter"
+      />
+      {isTitleLevelRequestsFeatureEnabled && (
+        <MediatedRequestLevelFilter
+          data-testid="mediatedRequestFiltersLevelFilter"
+        />
+      )}
+    </form>
+  );
+};
+
+MediatedRequestsFilters.manifest = {
+  configs: {
```

Comment:
Please use `react-query` instead of `stripes-connect` for API queries. New code should be written with `react-query`

Reply:
Fixed